src/pages/tasks/NewTaskPage.js

```diff
@@ -153,15 +153,15 @@ const NewTaskPage = (props) => {
                     <MenuItem
                         label={assignee.displayName ? props.translate('newTaskPage.assignee') : ''}
                         title={assignee.displayName || ''}
-                        description={assignee.subtitle ? assignee.subtitle : props.translate('newTaskPage.assignee')}
+                        description={assignee.displayName ? assignee.subtitle : props.translate('newTaskPage.assignee')}
```

Comment:
This does not make sense. Did you mean to set the description to `assignee.displayName` (currently it's still `assignee.subtitle`)

Reply:
Currently the email is not shown. Can you please add that

Reply:
yeah let's fix this here assuming it's not much trouble